pkg/apis/rke.cattle.io/v1/zz_generated_deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
If this is the only change the `go generate` produced, can we leave this out?

Reply:
Someone ran go generate with Go 1.17 which adds that line. I ran go generate with Go 1.16 (Go version stated in Rancher's go.mod) which then removed this line. I kept it because it's changing it back to how it's supposed to be.